### ISSUE TITLE
Investigate components using `aria-expanded=false` as a possible WCAG violation

### ISSUE THREAD

**Issue by gd2910** [mi6/ic-ui-kit#2107]
## Summary of the bug 
We currently have some WCAG violations being addressed in ticket [1912](https://github.com/mi6/ic-ui-kit/issues/1912). The purpose of this ticket is to investigate and fix other identical violations in other components IE
![image](https://github.com/mi6/ic-ui-kit/assets/122624946/4fd81825-e613-4953-bfea-a2f6275fe3eb)

This includes work to include cypress testing/uncommenting tests and regenerating snapshots for unit tests 

## 📝 Acceptance Criteria
A11y tests should pass.

**Comment by ad9242** [comment]
After researching here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded , determined that components are ok as `aria-expanded` can be `false`. The cases in the components are generally on `<button>` elements which are acceptable
